src/App.js

```diff
@@ -29,11 +29,34 @@ class App extends Component {
     this.setState(user);
   }
 
+  determineHeaderText = () => {
```

Comment:
We'll have to decide whether `determineHeaderText( )` and `determineLogButtonStatus( )` should exist here in `App` or in `Header.` We made the decision to put it here because we were unsure of how much logic should exist inside a functional component such as `Header.`